README.rst

```diff
@@ -57,7 +34,7 @@ Alternatively, one can set environment variables to use AMReX and PelePhysics re
 Dependencies
 ~~~~~~~~~~~~
 
-`PeleC` was created as a renamed, stripped down version of `Maui`, and is built on the `AMReX` library.  In the process, the Microphysics folder was extracted, and reorganized into a separate repository, `PelePhysics`.  
+`PeleC` was created as a renamed, stripped down version of `Maui`, and is built on the `AMReX` library.  In the process, the Microphysics folder was extracted, and reorganized into a separate repository, `PelePhysics`. PeleC requires the `SUNDIALS <https://github.com/LLNL/sundials>`_ ODE solver library.
```

Comment:
Should we remove this Maui stuff? Doesn't really feel relevant anymore... Just say that we depend on PelePhysics for transport, EOS, and chem?

Reply:
Agreed.